src/components/app/ProfileViewer.js

```diff
@@ -177,9 +178,6 @@ export const ProfileViewer = explicitConnect<{||}, StateProps, DispatchProps>({
     icons: getIconsWithClassNames(state),
     isBottomBoxOpen: getIsBottomBoxOpen(state),
   }),
-  mapDispatchToProps: {
-    returnToZipFileList,
-    invalidatePanelLayout,
-  },
+  mapDispatchToProps: { returnToZipFileList, invalidatePanelLayout },
```

Comment:
it looks like your IDE's formatting configuration changes things that you don't touch otherwise (see also the changes to Details above), and that are not necessary. While I don't really mind, this makes the patches more difficult to review.

Reply:
Yes, it's not intentional. It's hard to catch these file changes when I run the lint commands. I'll keep an eye out in the future.